### ISSUE TITLE
ctrl_dbus: ignoring return value of write

### ISSUE THREAD

**Issue by alfredh** [baresip/baresip#1358]
Ubuntu 20.04 with gcc 9.3.0

```make
$ make RELEASE=yes

modules/ctrl_dbus/ctrl_dbus.c: In function âon_handle_invokeâ:
modules/ctrl_dbus/ctrl_dbus.c:160:2: error: ignoring return value of âwriteâ, declared with attribute warn_unused_result [-Werror=unused-result]
  160 |  write(st->fd[1], buf, sizeof(buf));
      |  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
modules/ctrl_dbus/ctrl_dbus.c: In function âcommand_handlerâ:
modules/ctrl_dbus/ctrl_dbus.c:142:2: error: ignoring return value of âreadâ, declared with attribute warn_unused_result [-Werror=unused-result]
  142 |  read(st->fd[0], buf, sizeof(buf));
      |  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
modules/ctrl_dbus/ctrl_dbus.c: At top level:
cc1: error: unrecognized command line option â-Wno-shorten-64-to-32â [-Werror]
cc1: all warnings being treated as errors
make: *** [mk/mod.mk:49: build-x86_64/modules/ctrl_dbus/ctrl_dbus.o] Error 1
```

**Comment by cspiel1** [comment]
will fix this tomorrow.

**Comment by alfredh** [comment]
fixed in b62f9fc0871684f1a94c1e7da266cc67ac561c9a
